components/dashboard/Payments.js

```diff
@@ -43,6 +43,13 @@ function Payments ({ limit }) {
     const displayedAgreements = limit ? paidAgreements.slice(0, limit) : paidAgreements;
     return displayedAgreements.map((agreement, index) => (
       <div key={index} className={styles["payment-item"]}>
+        <div className={styles["payment-avatar"]}>
```

Comment:
the payments page has lost its central layout and everything is aligned to the left